ci/test-conformance-eks.sh

```diff
@@ -181,9 +185,14 @@ function deliver_antrea_to_eks() {
 
     set +e
     worker_node_ip_1=$(kubectl get nodes -o wide | sed -n '2 p' | awk '{print $7}')
-    node_mtu=$(ssh -o StrictHostKeyChecking=no -l "ec2-user" $worker_node_ip_1 \
+    node_mtu=$(ssh -i $SSH_PRIVATE_KEY_PATH -o StrictHostKeyChecking=no -l "ec2-user" $worker_node_ip_1 \
       'export PATH=$PATH:/usr/sbin; ip a | grep -E eth0.*mtu | cut -d " " -f5')
 
+    if [[ -z ${node_mtu} ]]; then
+         echo "Cannot determine node MTU!"
+         exit 1
+    fi
+
     if [[ -z ${GIT_CHECKOUT_DIR+x} ]]; then
```

Comment:
If we set `GIT_CHECKOUT_DIR` based on the relative path from this script, wouldn't that work for both Jenkins and local usage?
```
THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
GIT_CHECKOUT_DIR=$THIS_DIR/../..
```

Reply:
Done